lib/config.ts

```diff
@@ -0,0 +1,5 @@
+const config = {
+    NPM_TREE_SIZE_LIMIT: 6.0e6,
+    YARN_TREE_SIZE_LIMIT: 6.0e6,
+};
+export const {NPM_TREE_SIZE_LIMIT, YARN_TREE_SIZE_LIMIT} = config;
```

Comment:
we should use our config library for this https://github.com/snyk/config

Reply:
We cannot do that yet, requires further work in terms of `ts versioning` and I prefer create a separated pr for that

Reply:
We should use our config library in here, rater that adding more configs. What's the issue with `ts versioning` you mentioned above?

Reply:
This project in terms of config is still behind others and need certains upgrade that I prefer avoid on this pr since that they are not related.

Reply:
As I promised I created a side pr that solve the concerns that I had before of implement our config lib, now I have update this pr ⚡

Reply:
As I promised I created a side pr that solve the concerns that I had before of implement our config lib, now I have update this pr ⚡freely :)